front-end/src/App.tsx

```diff
@@ -93,7 +93,7 @@ const App = () => {
 													<Route exact path="/post/:id" >
 														<PostDiscussion/>
 													</Route>
-													<Route path="/onchain" >
+													<Route path="/on-chain" >
```

Comment:
changing link should not happen after app is live for long. There might we links to this page now on the interwebs which might be broken. If you want to change add a redirection /onchain -> /on-chain at least

Reply:
Fair enough, we'll keep this one then.